api/src/BcGov.Malt.Web/Startup.cs

```diff
@@ -57,23 +60,35 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddAuthentication(options =>
             {
-                options.DefaultAuthenticateScheme = Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerDefaults.AuthenticationScheme;
-                options.DefaultChallengeScheme = Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerDefaults.AuthenticationScheme;
+                options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
+                options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
             }).AddJwtBearer(o =>
             {
+                /* TODO Add keycloak authentication params once its ready .The following may change in future
+                 * For now , Jwt:Authority = the url to your local keycloak relam , eg: Master or ISB
+                 *               Audience = the name of the client you create in keycloak relam ,e.g: maltd or demo-app 
+                 *   "Jwt":{
+                             "Authority": "http://localhost:8080/auth/realms/<Relam_Name>",
+                             "Audience": "maltd"
+                } */
                 o.Authority = Configuration["Jwt:Authority"];
                 o.Audience = Configuration["Jwt:Audience"];
-                o.Events = new Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerEvents()
+                o.Events = new JwtBearerEvents()
                 {
                     OnAuthenticationFailed = c =>
                     {
                         c.NoResult();
 
-                        c.Response.StatusCode = 500;
+                        c.Response.StatusCode = 401;

```

Comment:
In .NET Core, we can use the new StatusCodes class, specifically the [Status401Unauthorized](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.http.statuscodes.status401unauthorized?view=aspnetcore-3.1) member.  It is more explicit and more clear in code.  We dont have to second guess if 401 is unauthorized or not found or forbidden, etc.  Change this on your next pass on this file.